### PR TITLE
zh: update restart tidbcluster document

### PR DESCRIPTION
**Pull request by Yisaer**
### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v1.1 changes for now)

Add restart document

### CODE REVIEW

File: zh/graceful-restart-tidb-component.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+title: 优雅重启 TidbCluster 组件弹性伸缩
```
Comment (ran-huang) [inline zh/graceful-restart-tidb-component.md]: Please update TOC.
Also, is `优雅重启` the official name? It sounds weird.

File: zh/graceful-restart-tidb-component.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+category: how-to
+---
+
+## 优雅重启 TidbCluster 组件弹性伸缩
+
+在使用 TiDB 集群的过程中，如果你发现某个集群节点存在内存泄漏等问题需要进行重启。我们可以通过优雅重启指令来将某个 TiDB 集群节点优雅下线然后再进行重新启动。
+
+> **警告：**
+>
+> 在生产环境未经过优雅重启，手动删除某个 TiDB 集群 Pod 节点是一件极其危险的事情，虽然 Stateful 控制器会将 Pod 节点再次拉起，但这依旧会对 TiDB 集群造成伤害。
```
Comment (DanielZhangQD) [inline zh/graceful-restart-tidb-component.md]: Can we change the word `伤害`？

File: zh/graceful-restart-tidb-component.md (hunk 1 above)
Comment (Yisaer) [reply zh/graceful-restart-tidb-component.md]: updated.

File: zh/graceful-restart-tidb-component.md (hunk 2 above)
Comment (Yisaer) [reply zh/graceful-restart-tidb-component.md]: updated

File: zh/graceful-restart-tidb-component.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
+---
+title: 优雅重启 TidbCluster 组件弹性伸缩
+category: how-to
+---
+
+## 优雅重启 TidbCluster 组件弹性伸缩
+
+在使用 TiDB 集群的过程中，如果你发现某个集群节点存在内存泄漏等问题需要进行重启。我们可以通过优雅重启指令来将某个 TiDB 集群节点优雅下线然后再进行重新启动。
```
[the remark this answers is not in the document]
Comment (Yisaer) [reply zh/graceful-restart-tidb-component.md]: updated.

File: zh/restart-a-tidb-cluster.md [hunk 4]
```diff
@@ -6,60 +6,39 @@ category: how-to
...
 # 重启 Kubernetes 上的 TiDB 集群
 
-本文描述了如何强制重启 Kubernetes 集群上的 TiDB 集群，包括重启某个 Pod，重启某个组件的所有 Pod 和重启 TiDB 集群的所有 Pod。
+在使用 TiDB 集群的过程中，如果你发现某个集群 Pod 节点存在内存泄漏等问题需要进行重启。本文描述了如何通过优雅重启指令来将某个 TiDB 集群 Pod 节点优雅下线然后再进行重新启动。
 
-> **注意：**
+> **警告：**
 >
-> TiDB Operator v1.0.x 版本只支持强制重启 Pod。
+> 在生产环境未经过优雅重启，手动删除某个 TiDB 集群 Pod 节点是一件极其危险的事情，虽然 Stateful 控制器会将 Pod 节点再次拉起，但这依旧可能会引起 TiDB 集群短暂的不可用。
```
Comment (DanielZhangQD) [inline zh/restart-a-tidb-cluster.md]:
```suggestion
> 在生产环境未经过优雅重启，手动删除某个 TiDB 集群 Pod 节点是一件极其危险的事情，虽然 StatefulSet 控制器会将 Pod 节点再次拉起，但这依旧可能会引起部分访问 TiDB 集群的请求失败。
```

File: zh/restart-a-tidb-cluster.md [hunk 5]
```diff
@@ -6,60 +6,39 @@ category: how-to
...
 
-```shell
-kubectl get pod -n <namespace> -l app.kubernetes.io/component=<component-name>
-```
 
-要强制重启某个组件的所有 Pod，执行以下命令：
+## 使用 annotate 标记目标 Pod 节点
 
-{{< copyable "shell-regular" >}}
+我们通过 `kubectl annotate` 的方式来标记目前 TiDB 集群 Pod 节点组件，当 `annotate` 标记完成以后，operator 会自动进行 Pod 节点的优雅下线并重启。你可以通过以下方式来进行标记:
```
Comment (DanielZhangQD) [inline zh/restart-a-tidb-cluster.md]:
```suggestion
我们通过 `kubectl annotate` 的方式来标记目前 TiDB 集群 Pod 节点组件，当 `annotate` 标记完成以后，TiDB Operator 会自动进行 Pod 节点的优雅下线并重启。你可以通过以下方式来进行标记:
```

File: zh/restart-a-tidb-cluster.md [hunk 6]
```diff
@@ -6,60 +6,39 @@ category: how-to
 
 # 重启 Kubernetes 上的 TiDB 集群
 
-本文描述了如何强制重启 Kubernetes 集群上的 TiDB 集群，包括重启某个 Pod，重启某个组件的所有 Pod 和重启 TiDB 集群的所有 Pod。
+在使用 TiDB 集群的过程中，如果你发现某个集群 Pod 节点存在内存泄漏等问题需要进行重启。本文描述了如何通过优雅重启指令来将某个 TiDB 集群 Pod 节点优雅下线然后再进行重新启动。
```
Comment (ran-huang) [inline zh/restart-a-tidb-cluster.md]:
```suggestion
在使用 TiDB 集群的过程中，如果你发现某个集群 Pod 节点存在内存泄漏等问题，需要对集群进行重启。本文描述了如何通过优雅重启指令来将某个 TiDB 集群 Pod 节点优雅下线然后再进行重新启动。
```

File: zh/restart-a-tidb-cluster.md [hunk 7]
```diff
@@ -6,60 +6,39 @@ category: how-to
...
 
-> **注意：**
+> **警告：**
 >
-> TiDB Operator v1.0.x 版本只支持强制重启 Pod。
+> 在生产环境未经过优雅重启，手动删除某个 TiDB 集群 Pod 节点是一件极其危险的事情，虽然 StatefulSet 控制器会将 Pod 节点再次拉起，但这依旧可能会引起部分访问 TiDB 集群的请求失败。
```
Comment (ran-huang) [inline zh/restart-a-tidb-cluster.md]:
```suggestion
> 在生产环境中，未经过优雅重启而手动删除某个 TiDB 集群 Pod 节点是一件极其危险的事情，虽然 StatefulSet 控制器会将 Pod 节点再次拉起，但这依旧可能会引起部分访问 TiDB 集群的请求失败。
```

File: zh/restart-a-tidb-cluster.md [hunk 8]
```diff
@@ -6,60 +6,39 @@ category: how-to
...
+      create: true
+      mutation:
+        pods: true
+    ```
 
-通过以下命令可以列出组件目前有哪些 Pod：
+2. 安装/更新 operator
 
-{{< copyable "shell-regular" >}}
+    修改完 `values.yaml` 文件中上述配置项以后进行 TiDB-Operator 部署或者更新。安装与更新 Operator 请参考[在 Kubernetes 上部署 TiDB Operator](deploy-tidb-operator.md)。
```
Comment (ran-huang) [inline zh/restart-a-tidb-cluster.md]:
```suggestion
    修改完 `values.yaml` 文件中的上述配置项以后，进行 TiDB-Operator 部署或者更新。安装与更新 Operator 请参考[在 Kubernetes 上部署 TiDB Operator](deploy-tidb-operator.md)。 
```

File: zh/restart-a-tidb-cluster.md [hunk 9]
```diff
@@ -6,60 +6,39 @@ category: how-to
...
 
-本文描述了如何强制重启 Kubernetes 集群上的 TiDB 集群，包括重启某个 Pod，重启某个组件的所有 Pod 和重启 TiDB 集群的所有 Pod。
+在使用 TiDB 集群的过程中，如果你发现某个集群 Pod 节点存在内存泄漏等问题，需要对集群进行重启。本文描述了如何通过优雅重启指令来将某个 TiDB 集群 Pod 节点优雅下线然后再进行重新启动。
 
-> **注意：**
+> **警告：**
 >
-> TiDB Operator v1.0.x 版本只支持强制重启 Pod。
+> 在生产环境中，未经过优雅重启而手动删除某个 TiDB 集群 Pod 节点是一件极其危险的事情，虽然 StatefulSet 控制器会将 Pod 节点再次拉起，但这依旧可能会引起部分访问 TiDB 集群的请求失败。
 >
```
Comment (DanielZhangQD) [inline zh/restart-a-tidb-cluster.md]: Remove this line.

File: zh/restart-a-tidb-cluster.md [hunk 10]
```diff
@@ -6,60 +6,39 @@ category: how-to
...
-要强制重启某个组件的所有 Pod，执行以下命令：
+## 使用 annotate 标记目标 Pod 节点
 
-{{< copyable "shell-regular" >}}
+我们通过 `kubectl annotate` 的方式来标记目前 TiDB 集群 Pod 节点组件，当 `annotate` 标记完成以后，TiDB Operator 会自动进行 Pod 节点的优雅下线并重启。你可以通过以下方式来进行标记:
 
-```shell
-kubectl delete pod -n <namespace> -l app.kubernetes.io/component=<component-name>
-```
+    {{< copyable "shell-regular" >}}
```
Comment (DanielZhangQD) [inline zh/restart-a-tidb-cluster.md]: No need to indent here.

File: zh/restart-a-tidb-cluster.md (hunk 9 above)
Comment (Yisaer) [reply zh/restart-a-tidb-cluster.md]: updated.

File: zh/restart-a-tidb-cluster.md (hunk 10 above)
Comment (Yisaer) [reply zh/restart-a-tidb-cluster.md]: updated.

File: zh/restart-a-tidb-cluster.md [hunk 11]
```diff
@@ -6,60 +6,38 @@ category: how-to
...
-把 `<component-name>` 分别替换为 `pd`、`tidb`、`tikv`，可以分别强制重启 `PD`、`TiDB`、`TiKV` 组件所有 Pod。
+    ```yaml
+    admissionWebhook:
+      create: true
+      mutation:
+        pods: true
+    ```
 
-## 强制重启 TiDB 集群的所有 Pod
+2. 安装/更新 operator
```
Comment (DanielZhangQD) [inline zh/restart-a-tidb-cluster.md]:
```suggestion
2. 安装/更新 TiDB Operator
```

File: zh/restart-a-tidb-cluster.md [hunk 12]
```diff
@@ -6,60 +6,38 @@ category: how-to
...
 
-通过以下命令可以列出 TiDB 集群目前有哪些 Pod，包括 `monitor`、`discovery` 等：
+    修改完 `values.yaml` 文件中的上述配置项以后，进行 TiDB-Operator 部署或者更新。安装与更新 Operator 请参考[在 Kubernetes 上部署 TiDB Operator](deploy-tidb-operator.md)。
```
Comment (DanielZhangQD) [inline zh/restart-a-tidb-cluster.md]:
```suggestion
    修改完 `values.yaml` 文件中的上述配置项以后，进行 TiDB Operator 部署或者更新。安装与更新 TiDB Operator 请参考[在 Kubernetes 上部署 TiDB Operator](deploy-tidb-operator.md)。 
```

File: zh/restart-a-tidb-cluster.md [hunk 13]
```diff
@@ -6,60 +6,38 @@ category: how-to
...
 
-{{< copyable "shell-regular" >}}
 
-```shell
-kubectl get pod -n <namespace> -l  app.kubernetes.io/instance=<tidb-cluster-name>
-```
+## 使用 annotate 标记目标 Pod 节点
 
-要强制重启 TiDB 集群的所有 Pod，包括 `monitor`、`discovery` 等，执行以下命令：
+我们通过 `kubectl annotate` 的方式来标记目前 TiDB 集群 Pod 节点组件，当 `annotate` 标记完成以后，TiDB Operator 会自动进行 Pod 节点的优雅下线并重启。你可以通过以下方式来进行标记:
```
Comment (DanielZhangQD) [inline zh/restart-a-tidb-cluster.md]:
```suggestion
我们通过 `kubectl annotate` 的方式来标记目标 TiDB 集群 Pod 节点组件，当 `annotate` 标记完成以后，TiDB Operator 会自动进行 Pod 节点的优雅下线并重启。你可以通过以下方式来进行标记:
```

**Pull request merged**